pkg/reader/ccip_test.go

```diff
@@ -624,7 +612,7 @@ func TestCCIPChainReader_DiscoverContracts_HappyPath_OnlySupportDest(t *testing.
 	destExtended.EXPECT().ExtendedGetLatestValue(
```

Comment:
Do you think we could create a method to factorize the ExtendedGetLatestValue? It's repeated quite a lot in this file.

Something like:

```
func setupMockExpectations(mock *MockExtended, destChain cciptypes.ChainSelector, destFeeQuoter, destRouter []byte) {
    mock.EXPECT().ExtendedGetLatestValue(
        mock.Anything,
        consts.ContractNameOffRamp,
        consts.MethodNameOffRampGetDynamicConfig,
        primitives.Unconfirmed,
        map[string]any{},
        mock.Anything,
    ).Return(nil).Run(withReturnValueOverridden(func(returnVal interface{}) {
        v := returnVal.(*offRampDynamicChainConfig)
        v.FeeQuoter = destFeeQuoter
        v.Router = destRouter
    }))
}
```